NEWS.md

```diff
@@ -1,3 +1,12 @@
+# tic 0.2.13.9011
+
+## step_rcmdcheck()
```

Comment:
Please do not update the NEWS file, I'm happy to take suggestions for NEWS entries in a comment.

Reply:
Ok!